Patches/MeetingHudPatch.cs

```diff
@@ -1188,6 +1190,10 @@ public static void Postfix(MeetingHud __instance)
             sb.Append(Romantic.TargetMark(seer, target));
             sb.Append(Lawyer.LawyerMark(seer, target));
 
+            //病娇提示
+            if (seer.Is(CustomRoles.Yandere))
+            sb.Append(Yandere.GetTargetArrow(seer));
+
             //会議画面ではインポスター自身の名前にSnitchマークはつけません。
```

Comment:
Move this code into 

`case CustomRoles.Yandere:
      break;`

Listed above this post